basalt/autograd/ops/dynamics.mojo

```diff
@@ -32,7 +32,13 @@ struct CONCAT:
     @staticmethod
     fn forward[
         attributes: AttributeVector,
-    ](inputs: List[Symbol], outputs: List[Symbol]):
+        mutability: __mlir_type.i1,
+        lifetime: AnyLifetime[mutability].type,
+    ](
+        inputs: List[Symbol],
+        outputs: List[Symbol],
+        parameters: Reference[Parameters, mutability, lifetime],
+    ):
```

Comment:
Is it necessary to use reference here?, isn't it the same to use borrowed?

Reply:
Damn, yes ofcourse..., making thing unnecessarily complicated here

Reply:
Fixed